cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -139,6 +137,8 @@ public static ConsentLibBuilder newBuilder(Integer accountId, String property, I
         onConsentUIFinished = b.onConsentUIFinished;
         shouldCleanConsentOnError = b.shouldCleanConsentOnError;
 
+        mCountDownTimer = getTimer(defaultMessageTimeOut);
```

Comment:
@leodoin We should initialize timer after the initialization of defaultMessagTimeout field.

Reply:
Thanks for spotting this @Suryakantj . I'm gonna actually pass ConsentLibBuilder. defaultMessageTimeOut as a param and get rid of this unnecessary state once and for all!